docs/03-for-team-members/02-becoming-a-team-member.md

```diff
@@ -0,0 +1,42 @@
+# Team members
+
+Flathub, as a centralized repository of software apps, has a dedicated team of
+contributors necessary to keep it running. These contributors are in charge of:
+
+* Reviewing new applications
+* Reviewing permission changes for existing aplications
+* Requesting permission changes for existent apps that might not fit the
+  guidelines
+
+In consequence, they can be seen as the auditors and moderators of our
+community. As such, they play a big role in the decision-making process.
+They are also an important part of the Flathub and Flatpak trust model. The fact
+that applications permissions are audited by the community is a warranty to
```

Comment:
```suggestion
that application's permissions are audited by the community is a warranty to
```

Reply:
"by the community" - maybe "by the moderators"?

Reply:
I guess it's up to you if you want to use one or the other. Applied the suggestion, but keep this open for the community/moderators discussion